pkg/agent/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -197,7 +198,17 @@ func newNetworkPolicyWithMultipleRules(name string, uid types.UID, from, to, app
 	}
 }
 
+func prepareFeatureTables() {
+	openflow.AntreaPolicyEgressRuleTable.SetOFTable(uint8(5), nil)
```

Comment:
Is it possible to provide a general test function from pkg/agent/openflow for static assigning ID to FeatureTable?

Reply:
@tnqn

Reply:
Updated.